Poker.BE/Poker.BE.Service/IServices/IAuthenticationService.cs

```diff
@@ -1,15 +1,20 @@
-﻿using System;
+﻿using Poker.BE.Service.Modules.Requests;
+using Poker.BE.Service.Modules.Results;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-
+//TODO @idan implement ipoker service
```

Comment:
@wainergal do you want me to take this? or not? let's talk about this change the trello accordingly. ❓